doc/man3/EVP_PKEY_CTX_ctrl.pod

```diff
@@ -701,6 +706,9 @@ EVP_PKEY_CTX_set_dsa_paramgen_md_props(), EVP_PKEY_CTX_set_dsa_paramgen_gindex()
 EVP_PKEY_CTX_set_dsa_paramgen_type() and EVP_PKEY_CTX_set_dsa_paramgen_seed()
 were added in OpenSSL 3.0.
 
+EVP_PKEY_CTX_set_group_name() and EVP_PKEY_CTX_get_group_name() were added in
```

Comment:
I think this would normally be merged with the one above i.e
a, b, c and d were added in OpenSSL 3.0.